types/passport-github/index.d.ts

```diff
@@ -1,33 +1,50 @@
-// Type definitions for passport-github 1.1
+// Type definitions for passport-github 1.2
```

Comment:
Version of the [underlying package](https://github.com/jaredhanson/passport-github) is still 1.1, so you should leave the version as is.

Reply:
```suggestion
// Type definitions for passport-github 1.2
```

I misunderstood this with the version. now its back again to 1.1 😃